api.js

```diff
@@ -84,18 +85,15 @@ class Api extends Emittery {
 		return new AvaFiles({cwd: apiOptions.resolveTestsFrom, files, extensions: this._allExtensions}).findTestFiles()
 			.then(files => {
 				if (this.options.parallelRuns) {
-					// The files must be in the same order across all runs, so sort them.
-					files = files.sort((a, b) => a.localeCompare(b, [], {numeric: true}));
-
 					const {currentIndex, totalRuns} = this.options.parallelRuns;
 					const fileCount = files.length;
-					const each = Math.floor(fileCount / totalRuns);
-					const remainder = fileCount % totalRuns;
 
-					const offset = Math.min(currentIndex, remainder) + (currentIndex * each);
-					const currentFileCount = each + (currentIndex < remainder ? 1 : 0);
+					// The files must be in the same order across all runs, so sort them.
+					files = files.sort((a, b) => a.localeCompare(b, [], {numeric: true}));
```

Comment:
Couldn't the sorting here be part of `chunkd` too? Maybe as an opt-in option.

Reply:
I need to be able to configure the comparator, but I guess it could do that too.